### DISCUSSION TITLE
日期选择器类型为monthrange，如何进行月范围限制

### DISCUSSION THREAD

**Discussion by pipiquan** [element-plus/element-plus#10175] [Q&A]
el-date-picker，type设置monthrange后，选择完第一个月份，然后想要对月份范围进行限制，例如，选择2022年5月后，月份范围选择缩减为前后2个月，应该如何实现？
日的选择可以通过calendar-change或者双向数据绑定的参数去抓起第一个选择的日，然后通过disabled-date去实现禁用选择的日，但是月份的无法触发calendar-change事件，对于双向绑定的数据也无法watch到。
请大佬们指点迷津。

**Comment by fanxingluochen** [comment]
同问
